bundles/core/org.eclipse.smarthome.core.persistence/META-INF/MANIFEST.MF

```diff
@@ -7,13 +7,20 @@ Bundle-ManifestVersion: 2
 Bundle-License: http://www.eclipse.org/legal/epl-v10.html
 Bundle-SymbolicName: org.eclipse.smarthome.core.persistence
 Import-Package: org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.common.registry,
  org.eclipse.smarthome.core.items,
  org.eclipse.smarthome.core.library.types,
  org.eclipse.smarthome.core.persistence,
  org.eclipse.smarthome.core.types,
+ org.quartz,

```

Comment:
I know :wink: 
I used quartz just temporary to not change to much code.
I will use the new scheduler implementation (the automation should move its timer trigger, too) after the concept of the split is okay.